README.md

```diff
@@ -2,7 +2,14 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 
 ## Getting Started
 
-First, run the development server:
+First,install dependencies
+
+```bash
+npm install
+# or
```

Comment:
Notice that we don't have a `package-lock.json` but just a `yarn.lock`. We don't use npm — only yarn.

So please include only instructions for yarn.